README.md

```diff
@@ -0,0 +1,78 @@
+# filecoin-docs
+Filecoin Docs
+
+
+<!--TO-DO: where can I get assets for Filecoin to replace IPFS logo and Build, Made, Project Icons?-->
+
+![IPFS documentation logo.](ipfs-docs-header.png)
+[![Build status icon.](https://img.shields.io/circleci/project/github/ipfs/ipfs-docs/master.svg?style=flat-square)](https://circleci.com/gh/ipfs/ipfs-docs)
```

Comment:
This should be generated automatically. @johnnymatthews do you know how this is connected to circleci?

Reply:
We don't need to use the _Build status icon_ (that's a legacy thing in IPFS docs that we should actually remove).

```suggestion
```